src/main/java/gov/nasa/pds/api/engineering/elasticsearch/ElasticSearchUtil.java

```diff
@@ -24,6 +26,14 @@ static public String jsonPropertyToElasticProperty(String jsonProperty) {
 		
 	}
 	
+	static private void append (ArrayList<Reference> to, String ID, String baseURL)
```

Comment:
Move the method after the following one, to keep elasticPropertyToJsonProperty and jsonPropertyToElasticProperty together

Reply:
Append ".orex" because that is what your acceptance criteria in 52 shows as being required.

If that is in error, remark here and I will remove it.